utils/types.ts

```diff
@@ -1,23 +1,15 @@
 export interface FoodPantry {
   name: string;
+  website: string;
+  email: string;
```

Comment:
Make these two properties optional since all Food Pantries may not have an email and website
```suggestion
  website?: string;
  email?: string;
```